devtools/platform-descriptor-json/src/main/resources/templates/dockerfile-jvm.ftl

```diff
@@ -14,21 +14,30 @@
 # docker run -i --rm -p 8080:8080 quarkus/${project_artifactId}-jvm
 #
 ###
-FROM fabric8/java-alpine-openjdk8-jre:1.6.5
-ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
-ENV AB_ENABLED=jmx_exporter
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.1
+
+ARG JAVA_PACKAGE=java-1.8.0-openjdk-headless
+ARG RUN_JAVA_VERSION=1.3.5
+
+# Install java and the run-java script
+# Also set up permissions for user `1001`
+RUN microdnf install ${JAVA_PACKAGE} \
+    && microdnf clean all \
+    && mkdir /deployments \
+    && chown 1001 /deployments \
+    && chmod "g+rwX" /deployments \
+    && chown 1001:root /deployments \
+    && curl https://repo1.maven.org/maven2/io/fabric8/run-java-sh/${RUN_JAVA_VERSION}/run-java-sh-${RUN_JAVA_VERSION}-sh.sh -o /deployments/run-java.sh \
+    && chown 1001 /deployments/run-java.sh \
+    && chmod 740 /deployments/run-java.sh \
```

Comment:
Shouldn't `550` work i.e. just executable and not modifiable by anyone? That would make the file immutable.

Reply:
Do we want the file to be executable by anyone from the group? 
540 seems ok, no?

* Executable by 1001
* Readable by the root group - but not executable
* Invisible for anyone else

Reply:
So should it be 540 or 740 as is in the code now?

Reply:
@geoand we are hesitating between 540 or 550 :-D

Reply:
My vote is for 540 since we generally try to make sure the process isn't run by root right?

Reply:
540 is it!